### DISCUSSION TITLE
希望回车可以代替点击发送按钮

### DISCUSSION THREAD

**Discussion by fajianren** [lencx/Noi#230] [General]
![image](https://github.com/user-attachments/assets/bbc0bdb2-aaa4-4dbc-b43d-6c2edeca6e4d)

**Comment by ssupinma** [comment]
我对ChatGPT最大的抱怨就是，他把回车 enter 默认为发送，而不给任何热键修改的可能。如果要把回车 enter 设定成发送，请给使用者可以自由设定，或是能让使用者用热键 alt-enter 取代 enter

**Comment by ZhenhongDu** [comment]
可以在软件里改的
![image](https://github.com/user-attachments/assets/646b66fd-ca88-4486-b24a-adbbb4064381)
